wear/src/main/java/io/homeassistant/companion/android/home/HomeActivity.kt

```diff
@@ -72,9 +79,11 @@ class HomeActivity : ComponentActivity(), HomeView {
             .inject(this)
 
         presenter.onViewReady()
+        mainScope.launch {
+            entityViewModel.entitiesResponse = presenter.getEntities()
+        }
```

Comment:
This code is reused in many places, it should be a function that can be called rather than repeated.